<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLineEdit" name="LineFirstName"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="ButtonResister">
     <property name="text">
      <string>Resister</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="LabelNum1">
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="LabelNum1_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="LinePasswordConfirm">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="LabelNum1_2">
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="LabelNum2">
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="LinePassword"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="LineLastName"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="ButtonCancel">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="LabelNum2_2">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="LineEmail"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="LabelNum2_3">
     <property name="text">
      <string>Password(confirm)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
